website/tutorial/interacting-with-a-database-server.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 5
+title: Interacting with a database server
+description: Interacting with databases from the Podman Desktop application
+keywords: [podman desktop, podman, databases]
+tags: [podman-desktop, interacting-with-databases]
+---
+
+# Interacting with a database server
+
+This tutorial covers the following end-to-end tasks required to interact with a database server from within the Podman Desktop UI:
+
+- Pulling a database server image
+- Creating a database server instance
+- Accessing the instance from terminal
+
+For creating a database instance, you can use one of the following options:
+
+- [Build a database server image using a container or docker file](/docs/containers/images/building-an-image)
+- [Import a database server image from your local machine](/tutorial/managing-your-application-resources#managing-images) using the **Import** button on the _Images_ component page
+- Pull a database server image from a registry (covered in this tutorial)
+
+This tutorial focuses on creating a PostgreSQL server instance by pulling the `quay.io/fedora/postgresql-16` image from the _quay.io_ registry.
+
+## Before you begin
+
+Make sure you have:
+
+- [Installed Podman Desktop](/docs/installation).
+- [A running Podman machine](/docs/podman/creating-a-podman-machine).
+- A developer role.
+
+## Pulling a PostgreSQL server image
+
+1. Click **Images** in the left navigation pane.
+1. Click the **Pull** button.
+   ![pulling an image](img/pull-button.png)
+1. Enter the name of the image to pull from the registry.
+1. Click **Pull image**. A download complete notification opens.
+   ![clicking the pull button](img/click-pull-image.png)
+1. Click **Done**.
+   ![image pulled successfully](img/image-successfully-pulled.png)
+1. View the newly created `quay.io/fedora/postgresql-16` image on the same page.
+   ![image added on the page](img/new-image-added.png)
+
+## Creating a PostgreSQL server instance
+
+1. Click **Images** in the left navigation pane.
+1. Click the **Run Image** icon corresponding to the PostgreSQL server image you want to run.
+   ![running an image](img/run-image-icon.png)
+1. Configure the basic details, such as container name, port mapping, and environment variables for the PostgreSQL server instance.
+1. Click **Start Container**.
+   ![starting a container](img/starting-a-postgresql-container.png)
+1. View the successful operation notification in the _Tty_ tab of the _Container Details_ page.
```

Comment:
I'm wondering why the Tty tab is used and not the Logs one ?

Reply:
Because when you perform the actual procedure, this tty tab automatically opens after you click `Start Container` (the previous step). The tty tab shows that the database server has started successfully.